ee/ucp/admin/configure/use-your-own-tls-certificates.md

```diff
@@ -30,33 +30,27 @@ will have to download new ones to [access UCP from the CLI](../../user-access/cl
 
 ## Configure UCP to use your own TLS certificates and keys
 
-In the UCP web UI, log in with administrator credentials and
-navigate to the **Admin Settings** page.
+To configure UCP to use your own TLS certificates and keys:
 
-  In the left pane, click **Certificates**.
+1. In the UCP web UI, log in with administrator credentials and
+navigate to the **Admin Settings** page.
 
-![](../../images/use-externally-signed-certs-2.png)
+2. Click **Certificates**.
 
-Upload your certificates and keys:
+    ![](../../images/ucp-certs.png)
```

Comment:
Could you:

- replace the existing file, instead of adding a new file? (i.e., name the new file the same as the old one: https://github.com/docker/docker.github.io/blob/master/ee/ucp/images/use-externally-signed-certs-2.png)
- looks like the image can be optimised losslessly to reduce size; optimised version is attached; could you replace your version with the optimised version? (I zipped it, otherwise GitHub shows it inline)

[ucp-certs.png.zip](https://github.com/docker/docker.github.io/files/4027002/ucp-certs.png.zip)